pymaker/deployment.py

```diff
@@ -219,7 +219,7 @@ def from_json(web3: Web3, conf: str):
                 else:
                     gem = DSToken(web3, Address(conf[name[1]]))
 
-                if name[1] in ['USDC', 'WBTC', 'TUSD']:
+                if name[1] in ['USDC', 'WBTC', 'TUSD', 'USDT']:
                     adapter = GemJoin5(web3, Address(conf[f'MCD_JOIN_{name[0]}']))
```

Comment:
I know TUSD and USDT are `GemJoin7`, not `GemJoin5`.  API is the same, but I included GemJoin5 for more accurate gas limit estimates.  PAXUSD uses plain ol' `GemJoin`.

Reply:
👍